src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/notifications/ReferralNotificationService.kt

```diff
@@ -102,6 +102,20 @@ class ReferralNotificationService(
         )
       }
 
+      ReferralEventType.DESIRED_OUTCOMES_AMENDED -> {
```

Comment:
Why this is added as part of your pull request?

Reply:
It has to be added as part of the when condition